assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,137 @@
 console.log('***** Music Collection *****')
+
+// Created a variable called collection and
+// the function addToCollection
+
+let collection = [];
+
+function addToCollection (title, artist, year) {
+    let album = {
+        title,
+        artist,
+        yearPublished: year,
+    };
+    collection.push(album)
+    return collection;
```

Comment:
Careful! The requirements specify that you must "Return the newly created object"!